package.json

```diff
@@ -55,13 +57,24 @@
     "serve": "gatsby serve",
     "deployStaging": "gatsby build && echo 'staging.openousd.org' > public/CNAME && gh-pages -d public",
     "promoteStaging": "./scripts/promote_staging.sh",
-    "refreshData": "python scripts/build_json.py"
+    "refreshData": "python scripts/build_json.py",
+    "percyStaging": "PERCY_TOKEN=$(grep PERCY_TOKEN .env.production | cut -d '=' -f2) PERCY_BRANCH=main percy snapshot --base-url=https://staging.openousd.org  scripts/testing/snapshots.js",
+    "percyProd": "PERCY_TOKEN=$(grep PERCY_TOKEN .env.production | cut -d '=' -f2) PERCY_BRANCH=main percy snapshot --base-url=https://openousd.org scripts/testing/snapshots.js"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/gatsbyjs/gatsby-starter-default"
   },
   "bugs": {
     "url": "https://github.com/gatsbyjs/gatsby/issues"
+  },
+  "percy": {
+    "version": 2,
+    "snapshot": {
+      "widths": [
+        1280,
+        375
```

Comment:
Where we specify we want mobile and desktop widths for each page. Browsers are set in the Percy console for some reason.